chai/__init__.py

```diff
@@ -1,3 +1 @@
-from chai.chai import Chai, NoServerConnection
-
-__all__ = ["Chai", "NoServerConnection"]
+from chai.chai import Chai, LoadModuleErr, NoServerConnection, RpcErr  # noqa
```

Comment:
Just curious about the `# noqa`, would the linter complain about not having `__all__` here or something else?

Reply:
It complains of unused modules without the `__all__`. :) If/when we have a more complex init, that does some set up and we need an `__all__`, we'll be able to remove it.

I initially only added an `__all__` here to quiet the linter, but having to enter everything twice to export it was getting on my nerves :D.